be/src/fs/fs_util.h

```diff
@@ -231,4 +231,8 @@ inline bool is_posix_uri(std::string_view uri) {
     return (memchr(uri.data(), ':', uri.size()) == nullptr) || starts_with(uri, "posix://");
 }
 
+inline bool is_jindo_uri(std::string_view uri) {
```

Comment:
so how do we use jindo now?

Reply:
can you put create catalog's sql in pr.

Reply:
It is no difference in catalog to  those of an ordinary s3/oss table.

The trick is more of a concept ambiguous.
Let me try to explain a bit more.

1.  jindo is yet another storage provide by alicloud/aliyun.
2. it use scheme  oss:// by default, just like the s3 service provided by alicloud/aliyun, but they are not the same thing.

so, to answer your 2nd question, there is no different from using like s3 table(in DDL/DML), no tricks here.

for the 1st question, you need to:
1. create a alicloud/aliyun account.
2. apply for a oss bucket.
3. enable oss-hdfs(alias for jindo) for that bucekt.
4. apply for a aliyun/alicoud machine to access using either api or command line tools aliyun/alicloud provided.

step 4 is required, as in their doc, the oss-hdfs/jindo service can only be access inside a aliyun/alicloud vpc network, not from a public.

Reply:
> can you put create catalog's sql in pr.

t is no difference in catalog to those of an ordinary s3/oss table.

The trick is more of a concept ambiguous.
Let me try to explain a bit more.

jindo is yet another storage provide by alicloud/aliyun.
it use scheme oss:// by default, just like the s3 service provided by alicloud/aliyun, but they are not the same thing.

so, to answer your 2nd question, there is no different from using like s3 table(in DDL/DML), no tricks here.

for the 1st question, you need to:

create a alicloud/aliyun account.
apply for a oss bucket.
enable oss-hdfs(alias for jindo) for that bucekt.
apply for a aliyun/alicoud machine to access using either api or command line tools aliyun/alicloud provided.

step 4 is required, as in their doc, the oss-hdfs/jindo service can only be access inside a aliyun/alicloud vpc network, not from a public.

Reply:
In case of how to configure the auth about jindo,
1. `oss://{key}:{secret}@${bucket}.${endpoint}` embedded in location url(path passed to fs).
2. merged options from `${STARROCKS_HOME}/conf/core-site.xml`, `${HADOOP_CONF_DIR}/core-site.xml`, `cloud_configuration`, `fs_option->hdfs_properties`
2.1 if `fs.oss.accessKeyId` and `fs.oss.accessKeySecret` exists, used as key&value
2.2 or if `fs.oss.bucket.${bucekt}.accessKeyId` and `fs.oss.bucket..${bucekt}..accessKeySecret` exists, used as key&value

Note, to avoid ambiguous, `fs.oss.accessKeyId` and `fs.oss.accessKeySecret` in core-site.xml wil be filtered out.
see `load_from_core_site`.

The main resolution can be found at `JindoFileSystem::resolve`